certified-connectors/VitaCloud Quotes/apiDefinition.swagger.json

```diff
@@ -0,0 +1,575 @@
+  "consumes": [],
+  "produces": [],
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://www.vitacloud.co.uk"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://www.vitacloud.co.uk/PrivacyPolicy"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Lifestyle and Entertainment;Productivity"
+    }
+  ],
+  "paths": {
+    "/Authors": {
+      "get": {
+        "tags": [
+          "Authors"
+        ],
+        "summary": "Get all authors",
+        "description": "Sample request:    GET /Authors",
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns the authors list",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/VwAuthor"
+              }
+            }
+          },
+          "204": {
+            "description": "If no list is returned"
+          }
+        },
+        "operationId": "GetAuthors"
+      }
+    },
+    "/Authors/{authortag}": {
+      "get": {
+        "tags": [
+          "Authors"
+        ],
+        "summary": "Get an author",
+        "description": "Sample request:    GET /Authors/aamilne",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "authortag",
+            "x-ms-summary": "Author Tag",
+            "x-ms-url-encoding": "single",
+            "description": "Author tag. For a list of available author tags, use the Get all authors action",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns the author",
+            "schema": {
+              "$ref": "#/definitions/VwAuthor"
+            }
+          },
+          "204": {
+            "description": "If no author is found for the supplied authortag"
+          }
+        },
+        "operationId": "GetAuthor"
+      }
+    },
+    "/Quotes/Today": {
+      "get": {
+        "tags": [
+          "Quotes"
+        ],
+        "summary": "Get today's quote",
+        "description": "Sample request:    GET /Quotes/today",
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns today's quote",
+            "schema": {
+              "$ref": "#/definitions/VwQuote"
+            }
+          },
+          "204": {
+            "description": "If no quote is returned"
+          }
+        },
+        "operationId": "GetTodaysQuote",
+        "x-ms-visibility": "important"
+      }
+    },
+    "/Quotes/Random": {
+      "get": {
+        "tags": [
+          "Quotes"
+        ],
+        "summary": "Get a random quote",
+        "description": "Sample request:\r\n    GET /Quotes/random",
```

Comment:
@kapilmehtagit please remove the extra white spaces and the special characters "\r\n" from the description. Please fix all the descriptions in the swagger file.